valhalla/mjolnir/landmark_builder.h

```diff
@@ -92,22 +92,69 @@ inline LandmarkType string_to_landmark_type(const std::string& s) {
 
 struct LandmarkDatabase {
 public:
+  /**
+   * LandmarkDatabase constructor.
+   * Create a new LandmarkDatabase object that connects to the SQLite landmark database
+   * with the given `db_name`. The `read_only` parameter determines whether the
+   * database connection is read-only or read-write.
+   * @param db_name The file path of the SQLite database to connect to.
+   * @param read_only Set to true to open the database in read-only mode, false for read-write.
```

Comment:
very nitpicky but we typically do this extra white space:

```suggestion
   * database connection is read-only or read-write.
   *
   * @param db_name    The file path of the SQLite database to connect to.
   * @param read_only  Set to true to open the database in read-only mode, false for read-write.
```

Reply:
done